src/ui/public/kfetch/kfetch.test.ts

```diff
@@ -30,15 +27,14 @@ jest.mock('../metadata', () => ({
   },
 }));
 
+import fetchMock from 'fetch-mock';
+import { kfetch } from './kfetch';
+
 describe('kfetch', () => {
-  const matcherName = /my\/path/;
+  const matcherName: any = /my\/path/;
```

Comment:
Can you use the `RegExp` type instead of `any`?

Reply:
.. or actually, i'd just ditch the type. Typescript will infer it since it is static

Reply:
The reason `any` is necessary is that the [types for `fetch-mock`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/fetch-mock/index.d.ts#L408) insist that the matcherName must be a string. By casting the variable to `any` we bypass the checks, and since the tests won't pass if the matcher isn't working correctly I feel okay about using `any` here.